src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -76,7 +78,7 @@ object DeployPlugin extends AutoPlugin {
       */
     val deployDirs = settingKey[Seq[String]](
       "subdirectories from the stage task to copy during deploy. " +
-        "defaults to bin/, conf/, lib/, and public/"
```

Comment:
More clearly: I would continue to sync all `conf` contents. Updating how `env.conf` is populated sounds awesome.